x-pack/plugin/sql/qa/server/src/main/java/org/elasticsearch/xpack/sql/qa/rest/RestSqlTestCase.java

```diff
@@ -498,6 +498,90 @@ public void testUseColumnarForTranslateRequest() throws IOException {
         }, containsString("unknown field [columnar]"));
     }
 
+    public void testValidateRuntimeMappingsInSqlQuery() throws IOException {
```

Comment:
If we move the validation of the RuntimeMappings in QL maybe we can have a simpler unit tests for the validation itself (incorrect json)